changelog.md

```diff
@@ -2,7 +2,9 @@
 
 ## UNRELEASED
 
-_Nothing yet..._
+**Added:**
+- bpk-svgs:
+  - Added icon font gulp task
```

Comment:
This should include description of what's changed from a consumer point of view - we haven't added a gulp task for them, we have added font to the package for consumer consumption.

Suggested:
```
- bpk-svgs:
  - Added ttf icon font
```